src/test/java/com/epam/ta/reportportal/ws/rabbit/AttachmentConsumerTest.java

```diff
@@ -64,4 +64,16 @@ void consume() {
 		verify(dataStoreService, times(1)).delete(attachment.getFileId());
 		verify(dataStoreService, times(1)).delete(attachment.getThumbnailId());
 	}
+
+	@Test
+	void consumePaths() {
+		ArrayList<String> paths = Lists.newArrayList("fileId");
+		DeleteAttachmentEvent event = new DeleteAttachmentEvent();
+		event.setPaths(paths);
+
+		attachmentConsumer.onEvent(event);
+
+		verify(dataStoreService, times(1)).delete(any());
+		verify(attachmentRepository, times(1)).deleteAllByIds(any());
```

Comment:
Better to use more specific check, i mean arguments from **delete(any())** and **deleteAllByIds(any())** change to specific values instead of **any()**
Could you change it, please.

Reply:
done

Reply:
ok, thx